### ISSUE TITLE
3.2.5 Example

### ISSUE THREAD

**Issue by kthang** [mschermann/data_viz_reader#892]
In 3.2.5, there is an example raised about interconnected networks.  It can be hard to follow what the example is talking about without some sort of visualization.  Ideally, there would be a before and after to show what focusing can accomplish.

**Comment by jacksonh2** [comment]
> Let’s consider the following example: You have visualized a graph and what came out of this was nothing but a mess of nodes connected through hundreds of edges (a very common result when visualizing so-called densely connected networks), one common transformation step would be to filter some of the edges. If, for instance, the edges represent money flows from donor countries to recipient countries, we could remove all flows below a certain amount (n.d.).

This is the section that @kthang is talking about.

unfortunately I wasn't able find any before and after graphs to demonstrate what focusing can do, but from the paragraph you can see that focusing (filtering in this case) is able to filter out the smaller amounts to get the big image on the money flows from donor countries to recipient countries, rather than focusing on every details and making the visualization messy.

**Comment by mschermann** [comment]
@kthang @jacksonh2 Can this be closed?

**Comment by mayjai** [comment]
I wasn't able to find the exact same before after chart for this example either, but I think I get the idea the paragraph was trying to convey simply by reading the words. So I think it should be fine if we leave it without providing example charts.
